
uint8 MODE
uint8 MODE_AUTO = 0
uint8 MODE_LOCK = 1
uint8 MODE_MAN = 2
uint8 MODE_SWITCH = 3       # chuyen trang thai